src/table/tableview/examples/table-view-basic.js

```diff
@@ -20,6 +20,7 @@
   * <ul style='list-style-type: none'>
   *   <li>.header     - (string) Text label for a column header
   *   <li>.itemField    - (string) Item field to associate with a particular column.
+  *   <li>.templateFn - (function) (optional) Template function used to render each cell of the column. Example: <pre>templateFn: value => `<span class="text-danger">${value}</span>`</pre>
   *   <li>.htmlTemplate - (string) (optional) id/name of an embedded ng/html template. Ex: htmlTemplate="name_template.html".  The template will be used to render each cell of the column.
```

Comment:
Same comment as above.  Please include some info as to why an app dev might choose htmlTemplate over templateFn, or visa-versa.  -thanks